src/ripple/app/misc/HashRouter.h

```diff
@@ -29,17 +29,20 @@
 
 namespace ripple {
 
-class STTx;
-
 // VFALCO NOTE Are these the flags?? Why aren't we using a packed struct?
 // VFALCO TODO convert these macros to int constants
 #define SF_RELAYED      0x01    // Has already been relayed to other nodes
 // VFALCO NOTE How can both bad and good be set on a hash?
-#define SF_BAD          0x02    // Signature/format is bad
-#define SF_SIGGOOD      0x04    // Signature is good
```

Comment:
Not so much removing it as hiding it / making it private so that callers are forced to go through `checkValidity`:
* https://github.com/ripple/rippled/pull/1230/files#diff-a8d97ad254f542a49802c0397343ea2cR43
* https://github.com/ripple/rippled/pull/1230/files#diff-e785a4713dff55ac6cd5320a7b295956R41